app/services/request_update_service.rb

```diff
@@ -149,7 +149,9 @@ def notify_request
 
   # complete the external approval process if configured
   def finish_request(decision)
-    return unless request.workflow.try(:external_processing?)
+    # PAM sends error and stops the process already
+    return if decision == Action::ERROR_OPERATION
```

Comment:
probably need a better indicator. If the error action was caused by other code and the PAM process is still running we still need to send the signal.
Why `request.process_ref.present?` is not sufficient? If PAM start failed with exception, we should not be able to record its `process_ref`.

Reply:
As discussed, we still can have `process_ref` since we turned on `Async` flag. Changed to use `request_complete?` to skip sending signal.